homeassistant/components/overkiz/climate/atlantic_electrical_towel_dryer.py

```diff
@@ -23,8 +23,9 @@
 
 OVERKIZ_TO_HVAC_MODE: dict[str, HVACMode] = {
     OverkizCommandParam.EXTERNAL: HVACMode.HEAT,  # manu
-    OverkizCommandParam.INTERNAL: HVACMode.AUTO,  # prog
-    OverkizCommandParam.STANDBY: HVACMode.OFF,
+    OverkizCommandParam.INTERNAL: HVACMode.AUTO,  # prog (schedule, user program)
+    OverkizCommandParam.AUTO: HVACMode.AUTO,  # auto (intelligent, user behavior)
```

Comment:
This will cause problems with the `HVAC_MODE_TO_OVERKIZ` mapping, as there is now a duplicate `AUTO`.

../Frenck

Reply:
This is actually a pattern we leverage more often in Overkiz integration (and it does work and not raise errors). Unfortunately internally the device do leverage more state params (OverkizCommandParam) for a similar state. 

You are correct though that this change will be a breaking change as the 'prog' mode cannot be selected in HA anymore, only the 'auto' mode. Let me see if I can make the overkiz 'prog' mode a preset, and map overkiz auto to hvac mode auto.